src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1049,16 +1049,7 @@ static bool matchFilter
 {
   /* First, look for unary operators */
   if ((opr == OPR_EXIST) || (opr == OPR_NOT_EXIST))
-  {
```

Comment:
Not really part of this PR, but a bit of cleanup regarding `+type` removal.

NTC